### DISCUSSION TITLE
Does the API return the shelf and compartment where items are located?

### DISCUSSION THREAD

**Discussion by OElesin** [Ephigenia/ikea-availability-checker#153] [Q&A]
Thanks for the amazing work with this API. I have two questions
1. Can I access shelf and compartment information from the API? Example if a product is in stock at a particular store, I would like to see the information where in the store I can find in the item.
2. Can I use the library within node or react application?

Thanks a lot for the help.

**Comment by Ephigenia** [comment]
> Can I access shelf and compartment information from the API? Example if a product is in stock at a particular store, I would like to see the information where in the store I can find in the item.

That information is not provided by the API. Maybe there’s another API that can be integrated.

> Can I use the library within node or react application?

Yes, just checkout the ["API" Section of the README](https://github.com/Ephigenia/ikea-availability-checker#api). There are also two simple examples in the [examples directory](https://github.com/Ephigenia/ikea-availability-checker/tree/master/examples).
